independent-publisher-connectors/Informa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1976 @@
+                                    },
+                                    "literal": {
+                                      "type": "string",
+                                      "description": "literal"
+                                    }
+                                  },
+                                  "description": "campoCodificadoFormaJuridicaAmpliada"
+                                },
+                                "fechaFuente": {
+                                  "type": "string",
+                                  "description": "Source Date",
+                                  "title": "Source Date"
+                                }
+                              },
+                              "description": "formaJuridica"
+                            },
+                            "capitalSocial": {
+                              "type": "object",
+                              "properties": {
+                                "importeCapitalSocial": {
+                                  "type": "number",
+                                  "format": "float",
+                                  "description": "Capital Stock Amount",
+                                  "title": "Capital Stock Amount"
+                                },
+                                "campoCodificadoTipoCapitalSocial": {
+                                  "type": "object",
+                                  "properties": {
+                                    "valor": {
+                                      "type": "string",
+                                      "description": "Value",
+                                      "title": "Value"
+                                    },
+                                    "tablaDecodificacion": {
+                                      "type": "string",
+                                      "description": "Decoding Table",
+                                      "title": "Decoding Table"
+                                    },
+                                    "literal": {
+                                      "type": "string",
+                                      "description": "literal"
+                                    }
+                                  },
+                                  "description": "campoCodificadoTipoCapitalSocial"
+                                },
+                                "fechaCapitalSocial": {
+                                  "type": "string",
+                                  "description": "Capital Stock Date",
+                                  "title": "Capital Stock Date"
+                                }
+                              },
+                              "description": "capitalSocial"
+                            }
+                          },
+                          "description": "datosGenerales"
+                        },
+                        "direcciones": {
```

Comment:
Please correct the spelling "direcciones"

Reply:
I think I should use 'formato' which is what the API expects. Is in spanish.

Reply:
Kindly post your connector in English, this will be translated to other languages later.

Reply:
The API requires specific parameters, as indicated. This is a Spanish solution that provides public information about Spanish companies.

Reply:
The string is not formato, it is "direcciones". As per the public documentation link mentioned below, your connector, titles, descriptions, operations, summaries everything should be in English. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions